.github/workflows/toml-lint.yml

```diff
@@ -0,0 +1,18 @@
+name: TOML lint
```

Comment:
Can this go in https://github.com/spiraldb/actions

Reply:
sure